pkg/equinixmetal/actuator/actuator.go

```diff
@@ -0,0 +1,364 @@
+/*
```

Comment:
Mostly copy/paste from `pkg/vsphere/actuator/actuator.go`

Reply:
This is one of those pieces that we don't want to add to the in-cluster operator. As an example of what we would like, you can view the PRs that were done for IBMCloud (the first pass they made at this effectively implemented Passthrough mode, but their current - as of today - PR removes this Passthrough with support for fine-grained creds) https://github.com/openshift/cloud-credential-operator/pull/356